catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/receive/SQSMessageReceiver.scala

```diff
@@ -73,11 +73,11 @@ class SQSMessageReceiver @Inject()(
   }
 
   val miroTransformableStore =
-    new S3TypedObjectStore[MiroTransformable](s3StringStore)
+    new S3TypeStore[MiroTransformable](s3StringStore)
   val calmTransformableStore =
-    new S3TypedObjectStore[CalmTransformable](s3StringStore)
+    new S3TypeStore[CalmTransformable](s3StringStore)
   val sierraTransformableStore =
-    new S3TypedObjectStore[SierraTransformable](s3StringStore)
+    new S3TypeStore[SierraTransformable](s3StringStore)
```

Comment:
FYI, I’ll push this naming back into my branch – there’s no point introducing it in diff 1 and immediately changing it in diff 2. Should make this PR a lot smaller too.